"issue5808/pull" e2e test failing

The "issue5808" e2e test is failing for every PR.  The FAIL message is
```
                   --- FAIL: TestE2E/SEQ/PULL/ordered/issue5808/pull (0.24s)
```
and the corresponding error message is
```
=== RUN   TestE2E/SEQ/PULL/ordered/issue5808/pull
    regressions.go:63: Running command "/usr/local/bin/singularity pull --arch amd64 --library https://library.sylabs.io /tmp/issue58081092110446/alpine.sif library://sylabs/tests/signed:1.0.0"
    regressions.go:63: 
        "/usr/local/bin/singularity pull --arch amd64 --library https://library.sylabs.io /tmp/issue58081092110446/alpine.sif library://sylabs/tests/signed:1.0.0" output:
        FATAL:   Unable to get keyserver client configuration: error response from server: %!s(<nil>)
```

--- a/e2e/pull/pull.go
+++ b/e2e/pull/pull.go
@@ -652,7 +652,8 @@ func E2ETests(env e2e.TestEnv) testhelper.Tests {
 			t.Run("pullDisableCache", c.testPullDisableCacheCmd)
 
 			// Regressions
-			t.Run("issue5808", c.issue5808)
+			// Disable for now, see issue #6299
+			// t.Run("issue5808", c.issue5808)
 		}),
 	}
 }

